api/volume.go

```diff
@@ -299,6 +300,29 @@ func (s *Server) VolumeUpdateReplicaCount(rw http.ResponseWriter, req *http.Requ
 	return s.responseWithVolume(rw, req, "", v)
 }
 
+func (s *Server) VolumeUpdateReplicaAutoBalance(rw http.ResponseWriter, req *http.Request) error {
+	var input UpdateReplicaAutoBalanceInput
+	id := mux.Vars(req)["name"]
+
+	apiContext := api.GetApiContext(req)
+	if err := apiContext.Read(&input); err != nil {
+		return errors.Wrapf(err, "error reading recurringInput")
```

Comment:
Wrong message, it should be something like `error reading replicaAutoBalance` (please help check others to fix as well if there are)

Reply:
Thanks! fixed.